lib/badfruit/Movies/movie.rb

```diff
@@ -26,6 +26,11 @@ def full_cast
       return @badfruit.parse_actors_array(JSON.parse(@badfruit.get_movie_info(@id, "cast")))
     end
 
+    # Returns director, since RottenTomatoes API is broken (abridged_director doesn't show up in search_by_name results for movies.)
+    def director
+      return Director.new(JSON.parse(@badfruit.get_movie_info(@id, "details"))["abridged_directors"][0])
```

Comment:
Actually, thinking about it more, I don't think we should cache the result value. Looking at the rest of the methods I specifically don't cache the result value. So I'd say what you have here is okay, but it needs to account for multiple directors.